composables/useAuth.ts

```diff
@@ -1,9 +1,12 @@
+import { useIdentityStore } from '@/stores/identity'
+
 export default function () {
-  const { $store } = useNuxtApp()
+  // const { $store } = useNuxtApp()
+  const identityStore = useIdentityStore()
 
-  const accountId = computed(() => $store.getters.getAuthAddress)
-  const isLogIn = computed(() => Boolean($store.getters.getAuthAddress))
-  const balance = computed(() => $store.getters.getAuthBalance)
+  const accountId = computed(() => identityStore.auth.address)
+  const isLogIn = computed(() => Boolean(identityStore.auth.address))
+  const balance = computed(() => identityStore.getAuthBalance)
```

Comment:
does this work without being passed urlPrefix?

Reply:
fixed by removing prefix arg